src/components/ToggleMenu.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import { Link } from 'react-router-dom';
+import MenuIcon from '@material-ui/icons/Menu';
+import HomeIcon from '@material-ui/icons/Home';
+import BarChartIcon from '@material-ui/icons/BarChart';
+import PersonPinIcon from '@material-ui/icons/PersonPin';
+import Tab from '@material-ui/core/Tab';
+
+export default function SimpleMenu() {
+    const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+
+    const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
Great job on adding the type for `event`! This is a tricky one for most beginners 👍